tensorzero-internal/tests/e2e/providers/together.rs

```diff
@@ -27,18 +27,19 @@ async fn get_providers() -> E2ETestProviders {
         credentials,
     }];
 
-    let json_providers = vec![
-        E2ETestProvider {
-            variant_name: "together".to_string(),
-            model_name: "llama3.1-8b-instruct-together".into(),
-            model_provider_name: "together".into(),
-            credentials: HashMap::new(),
-        },
-        // TODOs (#80): see below
-        // E2ETestProvider {
-        //     variant_name: "together-implicit".to_string(),
-        // },
-    ];
```

Comment:
would we want to also support `implicit_tool` mode for together now that tools are supported to some extent? It does seem that since `tool_choice=required` doesn't work this won't be possible at the moment